src/start/RealmScreen.js

```diff
@@ -68,13 +80,17 @@ class RealmScreen extends PureComponent<Props, State> {
 
     return (
       <Screen title="Welcome" padding scrollView>
-        <Label text="Your server URL" />
+        <Label
+          text="Register or log in to a Zulip organization to get started."
+          style={[styles.heading2, customStyles.heading]}
+        />
+        <Label text="URL of Zulip organization" />
         <Input
           style={styles.smallMarginTop}
           autoFocus
           autoCorrect={false}
           autoCapitalize="none"
-          placeholder="Server URL"
+          placeholder="your-organization.zulipchat.com"
```

Comment:
Also please add it `zulip-mobile/src/i18n/translations/messages_en.json`

Reply:
@jainkuniya OK, I added it to `zulip-mobile/src/i18n/translations/messages_en.json`. Thanks!